services/api/db/urls.py

```diff
@@ -0,0 +1,20 @@
+from django.urls import path
+from db.views import ProfileViewSet, OrganizationSettingsViewSet
+from rest_framework.urlpatterns import format_suffix_patterns
+
+
+urlpatterns = [
+    path('bio/', ProfileViewSet.as_view({'get': 'bio', 'post': 'bio', 'put': 'bio', 'delete': 'bio'})),
```

Comment:
should the name be added for urls?

Reply:
yes this API endpoint is necessary for updating fields so provide exact field name in URLs gives more readability to API user.

Reply:
Are we adding the name here in this PR?